spec/components/shared/searchable_collection_component_spec.rb

```diff
@@ -0,0 +1,69 @@
+require "rails_helper"
+
+RSpec.describe Shared::SearchableCollectionComponent, type: :component do
+  let(:form) { instance_double(GOVUKDesignSystemFormBuilder::FormBuilder) }
+
+  let(:collection) do
+    [1, 2, 3, 4, 5].freeze
+  end
+
+  before do
+    allow(form).to receive(:govuk_collection_radio_buttons)
+    allow(form).to receive(:govuk_collection_check_boxes)
+  end
+
+  context "Radio button searchable collection" do
```

Comment:
Our convention is that we either use `context "when(/with/without) something is the case"`, or `describe "feature/object/method"`. This comes from https://www.betterspecs.org/